app/views/LocationShare/services/liveLocation.ts

```
@@ -0,0 +1,150 @@
+	coords: {
+		latitude: number;
+		longitude: number;
+		accuracy?: number;
+	};
+	timestamp: number;
+	isActive: boolean;
+};
+
+export class LiveLocationTracker {
+	private watchSub: Location.LocationSubscription | null = null;
+	private tickInterval: NodeJS.Timeout | null = null;
```

ESLint

[error] 'NodeJS' is not defined.

(no-undef)